### PR TITLE
Add curl command into docker image

### PR DESCRIPTION
**Pull request by mikechengwei**
### What problem does this PR solve?

Fix tidb-operator pd health check question,ref :https://github.com/pingcap/tidb-operator/issues/4764.

### What is changed and how does it work?

```commit-message
Add curl command into docker image.
```

### Check List

Tests

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
- No code

Code changes

- Has configuration change
- Has HTTP API interfaces changed (Don't forget to [add the declarative for the new API](https://github.com/tikv/pd/blob/master/docs/development.md#updating-api-documentation))
- Has persistent data change

Side effects

- Possible performance regression
- Increased code complexity
- Breaking backward compatibility

Related changes

- PR to update [`pingcap/docs`](https://github.com/pingcap/docs)/[`pingcap/docs-cn`](https://github.com/pingcap/docs-cn):
- PR to update [`pingcap/tiup`](https://github.com/pingcap/tiup):
- Need to cherry-pick to the release branch

### Release note

```release-note
Please add a release note.

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

If you don't think this PR needs a release note then fill it with None.
```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -33,6 +33,9 @@ COPY --from=builder /go/src/github.com/tikv/pd/bin/pd-ctl /pd-ctl
 COPY --from=builder /go/src/github.com/tikv/pd/bin/pd-recover /pd-recover
 COPY --from=builder /jq /usr/local/bin/jq
 
+RUN apk add --no-cache \
```
Comment (rleungx) [inline Dockerfile:36]: Seem we have already added it in line 7?

Comment (mikechengwei) [reply Dockerfile:36]: I think line 7 means builder image install curl, but real pd image not install curl command.

Comment (mikechengwei) [reply Dockerfile:36]: This is similar to the dockerfile of tidb, you can refer to it: https://github.com/pingcap/tidb/blob/6.2.0/Dockerfile

**Review by lhy1024 — APPROVED** [review]

**Review by nolouch — APPROVED** [review]

**Comment by mikechengwei** [comment]
> /merge

plz help me label `/merge` again.

**Pull request merged**